corrector.rb

```diff
@@ -0,0 +1,5 @@
+class Corrector
+  def correct_name(name)
+    name.capitalize[0..10].strip
```

Comment:
- [ ] It will be great to trim the name when the characters are more than 10. There is a bug in this line of code. Remember when the name has more than 10 characters you should trim it. it is working perfectly the problem is you are trimming when the characters are 11, not ten to fix this issue consider the following. Either:
`[0,10]` to use a comma instead of a range, or `[0..9]` use a range but remember it is zero-indexed.

Reply:
Ok Thanks sir

Reply:
This change has been done perfectly 👏🏿 👏🏿